docs/getting-started.md

```diff
@@ -48,6 +48,9 @@ You will need to be running Linux, macOS, or Windows, and have [Python](https://
 
     ```bash
     pipx install meltano
+    
+    #If you have multiple versions of Python installed, you can use a specific one with the --python arugment
+    #  pipx install meltano --python [path to desired Python exe]
```

Comment:
The line "If you have multiple [...]" should be moved outside of the codeblock, and then the `pipx install meltano --python [...]` line should be in its own codeblock beneath that.

Reply:
I was trying to follow other examples, but I like this a lot better.
Fixed.